plugins/org.python.pydev.ast/src/org/python/pydev/ast/interpreter_managers/InterpreterInfo.java

```diff
@@ -1631,7 +1631,7 @@ public void setEnvVariables(String[] env) {
             File condaPrefix = this.getCondaPrefix();
             if (condaPrefix == null) {
                 Log.log("Unable to find conda prefix for: " + this.getExecutableOrJar());
-            } else if (new File(condaPrefix, "/etc/conda/activate.d").exists()) {
+            } else if (new File(condaPrefix, "/").exists()) {
```

Comment:
I don't think you need the `"/"` there anymore (it should work with: `new File(condaPrefix)`).